lib/resources/r4/charge_item_create.yaml

```diff
@@ -0,0 +1,413 @@
+fields:
+
+  - name: resourceType
+    required: 'Yes'
+    type: string
+    description: The type of the FHIR resource.
+    action: create
+    example: |
+      {
+        "resourceType": "Parameters"
+      }
+    note: resourceType must be Parameters
+    url: https://hl7.org/fhir/r4/json.html#resources
+
+  - name: parameter
+    required: 'Yes'
+    type: List of BackboneElement
+    description: Parameters defined by the operation.
+    action: create
+    url: http://hl7.org/fhir/r4/parameters-definitions.html#Parameters.parameter
+
+  - name: transactionId
+    required: 'Yes'
+    type: uuid
+    description: A UUID to uniquely identify the create operation.
+    action: create
+    example: |
+      {
+        "parameter": [
+          {
+            "name": "transactionId",
+            "valueUuid": "urn:uuid:aa7623a0-0c7b-11ec-82a8-0242ac130003"
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>The value must be in the form of `valueUuid`</li>
+        <li>Only one `transactionId` parameter may be provided</li>
+      </ul>
+    url: http://hl7.org/fhir/r4/parameters-definitions.html#Parameters.parameter
+
+  - name: chargeItem
+    required: 'Yes'
+    type: resource
+    description: The ChargeItem to be created.
+    action: create
+    example: |
+      {
+        "parameter": [
+          "name": "chargeItem",
+          "resource": {
+            "resourceType": "ChargeItem"
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>Resource field is required and it must be of type ChargeItem</li>
+        <li>Only one `chargeItem` parameter may be provided</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/parameters-definitions.html#Parameters.parameter
+
+  - name: status
+    required: 'Yes'
+    type: code
+    description: The current state of the Charge.
+    action: create
+    example: |
+      {
+        "status": "unknown"
+      }
+    note: >
+      <ul>
+        <li>The status must be set to `unknown`</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.status
+
+  - name: subject
+    required: 'Yes'
+    type: Reference
+    description: The patient associated with the charge.
+    action: create
+    example: |
+      {
+        "subject": {
+          "reference": "Patient/12742419"
+        }
+      }
+    note: >
+      <ul>
+        <li>Must be a Patient reference</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.subject
+
+  - name: context
+    required: 'Yes'
+    type: Reference
+    description: The encounter associated with the charge.
+    action: create
+    example: |
+      {
+        "context": {
+          "reference": "Encounter/97953601"
+        }
+      }
+    note: >
+      <ul>
+        <li>Must be an Encounter reference</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.context
+
+  - name: occurrenceDateTime
+    required: 'No'
```

Comment:
I think occurrenceDateTime is required for the create service.

Reply:
changed required to yes https://github.com/cerner/fhir.cerner.com/pull/778/commits/051542b5d68755b7c524626b97685248a6b576ae#diff-2830ed2523973e82037a23daf274840e05a6c6e05b9f698fbd5f3e7fc24f57bbR117